src/v/cloud_storage/materialized_segments.h

```diff
@@ -106,7 +106,8 @@ class materialized_segments {
 
     /// List of segments and readers waiting to have their stop() method
     /// called before destruction
-    eviction_list_t _eviction_list;
+    eviction_list_t _pending;
```

Comment:
Can we name these something like "_eviction_pending" and "_eviction_in_flight"?  I find just "pending" a bit obscure.

Reply:
done